9.2/run-postgresql.sh

```diff
@@ -90,12 +107,16 @@ function initialize_database() {
 
 		# Allow connections from all hosts.
 		host all all all md5
+
+		# Allow replication connections from all hosts.
+		host replication all all md5
 	EOF
 
 	pg_ctl -w start
 	createuser "$POSTGRESQL_USER"
 	createdb --owner="$POSTGRESQL_USER" "$POSTGRESQL_DATABASE"
 	psql --command "ALTER USER \"${POSTGRESQL_USER}\" WITH ENCRYPTED PASSWORD '${POSTGRESQL_PASSWORD}';"
+	psql --command "ALTER USER \"${POSTGRESQL_USER}\" WITH REPLICATION ;"
```

Comment:
There are probably better ways to avoid shell/SQL injection.  Discussed
some time ago off-github.  If we are talking only about replication
itself, I'm afraid I can't say something is good or wrong, or dangerous -
especially in paas.  It usually depends on the context.

Somebody who will maintain such complicated scenarios like SR/failover
should design optimal configuration - which will be useful for every user
of the image.